### PR TITLE
Add a consent popup for Firefox

### PR DESCRIPTION
**Pull request by rougetimelord**
Fine, have your consent popup! Does that make you happy?

Probably should beautify the page a bit, but honestly, if having an ugly page is what it takes to get a working extension out to users I can live with it.

Thanks for all of the memories AMO team <3 Really feeling great about continuing to support your browser <3

### CODE REVIEW

**Comment by cooljeanius** [comment]
so, about those CI failures...

**Comment by rougetimelord** [comment]
> so, about those CI failures...

T~T Look I had a beautiful woman impatiently telling me to wrap it up so we could go get drinks, all good now

**Comment by cooljeanius** [comment]
screenshot of what the pop-up looks like?

**Comment by rougetimelord** [comment]
> screenshot of what the pop-up looks like?

It is ugly ugly, but I'd rather get a version that Firefox will approve out than worry about beautifying it first...
image

**Comment by rougetimelord** [comment]
> please make bullets left text aligned (but aligned in the center of the page), buttons should be on their own row at the bottom

Here's what the page looks like now:
image

Might need some work for super narrow screens (adding padding) but I got dinner to cook and eat

**Comment by kheina** [comment]
thank you! looks fine to me

**Review by cooljeanius — DISMISSED** [review]

File: src/background/index.ts [hunk 1]
```diff
@@ -67,6 +67,21 @@ api.storage.sync.onChanged.addListener(async items => {
 
 ConnectDb();
 
+const consentRequiredVersions = ['0.3.5']
+
+api.runtime.onInstalled.addListener( ({reason, previousVersion}) => {
+	/** @ts-ignore I hate this :)*/
```
Comment (cooljeanius) [inline src/background/index.ts:73]: maybe update the comment here to be a bit more informative?

**Review by cooljeanius — DISMISSED** [review]

**Review by cooljeanius — APPROVED** [review]

**Pull request merged**